Sources/TuistKit/Services/DumpService.swift

```diff
@@ -37,8 +37,6 @@ final class DumpService {
             encoded = try manifestLoader.loadConfig(at: projectPath.appending(component: Constants.tuistDirectoryName))
         case .template:
             encoded = try manifestLoader.loadTemplate(at: projectPath)
-        case .dependencies:
-            encoded = try manifestLoader.loadDependencies(at: projectPath)
```

Comment:
Shouldn't we dump the config we have in `PackageSettings` config we have in `Package.swift`

Reply:
Could you rephrase? I'm not sure I follow

Reply:
In tuist 3 I can do `tuist dump dependencies` and get a bump of the dependencies and their configuration. How would i do it in tuist 4?

Reply:
I will add a command to dump the `PackageSettings` model 👍

Reply:
done 🙌